jjb/dynamic/http-scale-test.yml

```diff
@@ -0,0 +1,251 @@
+- job:
+    block-downstream: false
+    block-upstream: false
+    builders:
+    - shell: |-
+        if [[ ! -d "${WORKSPACE}/perf-dept" ]]; then
+        git clone https://${GITHUB_TOKEN}@github.com/redhat-performance/perf-dept.git
+        fi
+        export PRIVATE_KEY=${WORKSPACE}/perf-dept/ssh_keys/id_rsa_perf
+        chmod 600 ${PRIVATE_KEY}
+        http_test_dir=/root/http-ci-tests
+        ssh_opts="-o StrictHostKeyChecking=no"
+
+        if test "$USE_PROXY" = "true" ; then
+        set -- $ssh_opts -o ProxyCommand='ssh -i $PRIVATE_KEY -W %h:%p $PROXY_USER@$PROXY_HOST'
+        else
+        set -- $ssh_opts
+        fi
+
+        ssh "$@" -i $PRIVATE_KEY $JUMP_HOST_USER@$JUMP_HOST <<_SSH_
+        if [[ $CONTAINERIZED == "true" ]]; then
+        if [[ -d "/root/scale-cd-jobs" ]]; then
+        rm -rf /root/scale-cd-jobs
+        fi
+        git clone https://github.com/chaitanyaenr/scale-cd-jobs.git /root/scale-cd-jobs
+        cd /root/scale-cd-jobs/build-scripts
+        ./controller.sh
+        else
+        rm -rf $http_test_dir
+        git clone https://github.com/jmencak/http-ci-tests.git
```

Comment:
We're obsoleting this repo in favour of https://github.com/openshift-scale/workloads, I believe.  This ihttps://github.com/openshift-scale/workloads/pull/1 is very likely to merge tomorrow.

Reply:
We can create a PR with the modifications, the plan is to get the existing templates into the repo and then update the existing templates for all the jobs to use workloads tooling. Will add some instructions around testing the templates on a containerized jenkins instance soon.